api/src/main/java/org/pmiops/workbench/cohortreview/mapper/ParticipantCohortAnnotationMapper.java

```diff
@@ -0,0 +1,31 @@
+package org.pmiops.workbench.cohortreview.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.model.DbParticipantCohortAnnotation;
+import org.pmiops.workbench.db.model.DbStorageEnums;
+import org.pmiops.workbench.model.ParticipantCohortAnnotation;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(
+    config = MapStructConfig.class,
+    uses = {CommonMappers.class, DbStorageEnums.class})
+public interface ParticipantCohortAnnotationMapper {
+
+  @Mapping(
+      target = "annotationValueEnum",
+      source = "dbParticipantCohortAnnotation.cohortAnnotationEnumValue.name")
+  @Mapping(
+      target = "annotationValueDate",
+      source = "annotationValueDate",
+      qualifiedByName = "dateToString")
+  ParticipantCohortAnnotation dbModelToClient(
+      DbParticipantCohortAnnotation dbParticipantCohortAnnotation);
+
+  @Mapping(target = "cohortAnnotationEnumValue", ignore = true)
+  @Mapping(target = "annotationValueDateString", source = "annotationValueDate")
+  @Mapping(target = "annotationValueDate", ignore = true)
```

Comment:
Should we create a stringToDate method to use here?

Reply:
This section of code is old and I need to review why the annotations have a `annotationValueDateString` and `annotationValueDate`. When i have some time I will review this and we can create a ticket to cleanup if needed.